src/client/src/containers/About/index.tsx

```diff
@@ -35,21 +39,29 @@ const About = ({ versions, intl }: Props) => {
   const { formatMessage } = intl;
   return (
     <div className={styles.container}>
-      <div className={styles.title}>{formatMessage(messages.about)}</div>
       <p className={styles.repo}>
         <a
           className={styles.link}
           target="_blank"
           href={WEB_TEMPLATE_STUDIO_LINKS.REPO}
           onKeyUp={keyUpHandler}
         >
-          Web Template Studio
+          {formatMessage(messages.visitRepo)}
+        </a>
+        <a
+          className={styles.link}
+          target="_blank"
```

Comment:
Add `rel="noreferrer noopener"` for tslint whenever you use `target="_blank`. Thanks @Tanya0609 :)

Reply:
Just added :)